config/default.yaml

```diff
@@ -119,6 +119,15 @@ executor:
 
         # Launcher container tag to use
         launchVersion: stable
+    jenkins:
+      # Configuration of Jenkins
+      jenkins:
```

Comment:
It has been a while since we've updated our `stable` and `latest` tags as well 😞